django_celery_beat/models.py

```diff
@@ -166,6 +167,42 @@ def period_singular(self):
         return self.period[:-1]
 
 
+@python_2_unicode_compatible
+class ClockedSchedule(models.Model):
+    """clocked schedule."""
+
+    clocked_time = models.DateTimeField()
+    enabled = models.BooleanField(default=True, editable=False)
```

Comment:
Please add verbose_name and help_text kwargs like the other models now use.

Reply:
Done